src/driver.ml

```diff
@@ -230,11 +232,27 @@ module Transform = struct
       ; preprocess_intf = None
       ; enclose_impl
       ; enclose_intf
+      ; instrument = None
       ; rules
       ; registered_at = Caller_id.get ~skip:[]
       }
 
   let partition_transformations ts =
+    let before_instrs, after_instrs, rest = 
+      List.fold_left ts ~init:([], [], []) ~f:(fun (bef_i, aft_i, rest) t -> 
+        let red_t = 
```

Comment:
What does `red_t` stands for?

Reply:
I meant `reduced_t` with it. I've written it out now.

Reply:
Ack, yh `reduced` is definitely clearer !